test-java-lib/src/main/java/com/uber/lib/unannotated/AndroidRecentlyAnnotatedClass.java

```diff
@@ -0,0 +1,51 @@
+package com.uber.lib.unannotated;
+
+import androidx.annotation.RecentlyNonNull;
+import androidx.annotation.RecentlyNullable;
+
+// don't actually check this code
+@SuppressWarnings("NullAway")
+public class AndroidRecentlyAnnotatedClass {
+
+  public final Object field;
+
+  public AndroidRecentlyAnnotatedClass(Object field) {
+    this.field = field;
+  }
+
+  public @RecentlyNullable Object getField() {
+    return field;
+  }
+
+  public static @RecentlyNullable Object returnsNull() {
+    return null;
+  }
+
+  public static void consumesObjectNonNull(@RecentlyNonNull Object o) {}
+
+  public static void consumesObjectUnannotated(Object o) {}
+
+  public void acceptsNonNull(@RecentlyNonNull Object o) {}
+
+  public void acceptsNonNull2(@RecentlyNonNull Object o) {}
```

Comment:
Why the duplicated method?

Reply:
This was all copy-pasted from `test-java-lib/src/main/java/com/uber/lib/unannotated/RestrictivelyAnnotatedClass.java`.  I can clean up both if you like.

Reply:
Those are used here: https://github.com/uber/NullAway/blob/master/nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java#L1396 

Where we want to test that overriding a method with the same signature in two different ways behaves as expected :) But they are unused in this PR. I guess an alternative is a test for overriding in this case too? Maybe that's better?

Reply:
Here I just deleted all the unused methods.  I think adding another override test is kind of overkill given how this is implemented